modules/telco-core-cpu-partitioning-and-performance-tuning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-cpu-partitioning-and-performance-tuning_{context}"]
+= CPU partitioning and performance tuning
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+CPU partitioning separates sensitive workloads from general-purpose tasks, interrupts, and driver work queues, improving performance and reducing latency.
+The CPUs allocated to those auxiliary processes are referred to as reserved in the following sections.
+In Hyper-Threaded systems, a CPU is one hyper-thread.
```

Comment:
Could this be: In a system with Hyper-Threading enabled, as in [the bullet below](https://github.com/openshift/openshift-docs/pull/88622/files#diff-ba2ef29209f2231699796abfd657a0a8bfdfe52be1016b65e65f195d2d97a885R20)? I think it sounds better.